tests/heap/test_heap.py

```diff
@@ -82,6 +82,35 @@ def test_malloc_chunk_command(start_binary):
     for name in chunk_types:
         assert results[name] == expected[name]
 
+    gdb.execute("continue")
+
+    # Print main thread's chunk from another thread
```

Comment:
The 'from another thread' part is because we will break on 'break_here' in the new thread that we will be joining, right?

Reply:
Might mention it here as well, just for more clarity/understanding

Reply:
Or/and add sanity check like `assert gdb.selected_thread().<number field if there is such> == 2`

Reply:
Yes I like the sanity check, I'll add one or failing that a comment explaining what's going on.